common-util/api/leaderboard.js

```diff
@@ -15,7 +15,7 @@ export const getLeaderboardList = async () => {
     .map((user, index) => ({
       ...user,
       // adding a unique key to each user
-      rowKeyUi: `${user.twitter_handle}-${user.discord_id}-${index}`,
+      rowKeyUi: `${user.wallet_address}-${user.discord_id}-${index}`,
```

Comment:
Thinking about it, I think it's OK to include the odd, unrelated small improvement because otherwise the codebase stays completely static. Would be helpful for reviews to highlight the core vs ancilliary changes though!